gnomad/utils/annotations.py

```diff
@@ -1138,3 +1207,301 @@ def hemi_expr(
+    vrs_dict = {
+        "_id": vrs_id,
+        "location": {
+            "_id": "to-be-defined",
+            "interval": {
+                "end": {"type": "Number", "value": int(vrs_end_value)},
+                "start": {
+                    "type": "Number",
+                    "value": int(vrs_start_value),
+                },
+                "type": "SequenceInterval",
+            },
+            "sequence_id": vrs_chrom_id,
+            "type": "SequenceLocation",
+        },
+        "state": {"sequence": vrs_state_sequence, "type": "LiteralSequenceExpression"},
+        "type": "Allele",
+    }
+
+    # Set Location ID via code provided by Kyle F.
+    location_dict = vrs_dict["location"]
+    location_dict.pop("_id")
+    location = ga4gh.vrs.models.SequenceLocation(**location_dict)
+    location_id = ga4gh.core._internal.identifiers.ga4gh_identify(location)
+    vrs_dict["location"]["_id"] = location_id
+
+    logger.info(vrs_dict)
+
+    # If vrs_only was passed, only return the above dict and stop.
+    if vrs_only:
+        return vrs_dict
+
+    # Create a list to then add the dictionaries for frequency reports for different ancestry groups to.
+    list_of_group_info_dicts = []
+
+    # Define function to return a frequency report dictionary for a given group
+    def _create_group_dicts(
+        variant_ht: hl.Table,
+        group_index: int,
+        group_id: str,
+        group_label: str,
+        vrs_id: str,
+    ) -> dict:
+        """
+        Return a dictionary for the frequency information of a given variant for a given subpopulation
+
+        :param variant_ht: Hail Table with only one row, only containing the desired variant.
+        :param group_index: Index of frequency within the 'freq' annotation containing the desired group.
+        :param group_id: String containing variant, pop, and sex (if requested). Example: "chr19-41094895-C-T.afr.XX".
+        :param group_label: String containing the full name of pop requested. Example: "African/African American".
+        :param vrs_id: String containing the VRS ID of the variant in ht_subpop.
+        :return: Dictionary containing VRS information (and genetic ancestry group if desired) for specified variant.
+
+        """
+
+        # Obtain frequency information for the specified variant
+        group_freq = variant_ht.freq[group_index]
+
+        # Dictionary to be returned containing information for a specified group
+        freq_record =  {
+                    "id": group_id,
+                    "type": "PopulationAlleleFrequency",
+                    "label": f"{group_label} Population Allele Frequency for {variant}",
+                    "focusAllele": vrs_id,
+                    "focusAlleleCount": group_freq["AC"].collect()[0],
+                    "locusAlleleCount": group_freq["AN"].collect()[0],
+                    "alleleFrequency": group_freq["AF"].collect()[0],
+                    "population": f"gnomad3:{group_id}",
```

Comment:
use name and version labels here

Reply:
yup sounds good, if we get feedback that it throws something off we can change it later